### PR TITLE
feat(config): add config for Ring Range Extender

### PR DESCRIPTION
**Pull request by WizKid**
Tested it with firmware version 1.7 and 2.4 with the US version of the extender.

Those where the only version of the firmware I could get my hands on. So it is possible that the extra parameters was added earlier than 2.4 but I can't check that.

### CODE REVIEW

File: packages/config/config/devices/0x0346/range_extender_gen1.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Battery Report Interval",
+			"description": "Default heartbeat check in time",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0346/range_extender_gen1.json]:
```suggestion
```

File: packages/config/config/devices/0x0346/range_extender_gen1.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+			"maxValue": 70,
+			"defaultValue": 70,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"$if": "firmwareVersion >= 2.4",
+			"label": "Supervision Timeout",
+			"description": "The number of seconds waiting for a supervision report in response to a supervision get from the sensor",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0346/range_extender_gen1.json]:
```suggestion
			"description": "How long to wait for a supervision report in response to a supervision get from the sensor",
```

File: packages/config/config/devices/0x0346/range_extender_gen1.json [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+			"maxValue": 50,
+			"defaultValue": 15,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"3": {
+			"$if": "firmwareVersion >= 2.4",
+			"label": "Supervision Retries",
+			"description": "Number of application level retries attempted for messages either not ACKed or messages encapsulated via supervision get that did not receive a report",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0346/range_extender_gen1.json]:
```suggestion
			"description": "Number of application level retries attempted for messages either not ACKed or Supervision encapsulated messages that did not receive a report",
```

File: packages/config/config/devices/0x0346/range_extender_gen1.json [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+			"maxValue": 5,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"4": {
+			"$if": "firmwareVersion >= 2.4",
+			"label": "Supervision Back Off",
+			"description": "The number base seconds used in the calculation for sleeping between retry messages",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0346/range_extender_gen1.json]:
```suggestion
			"description": "Used to calculate the delay between retried messages",
```

File: packages/config/config/devices/0x0346/range_extender_gen1.json [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+			"minValue": 1,
+			"maxValue": 60,
+			"defaultValue": 5,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"5": {
+			"$if": "firmwareVersion >= 2.4",
+			"label": "Supervision Battery Report (one in N)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0346/range_extender_gen1.json]:
```suggestion
			"label": "Supervision Battery Report Frequency",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Review by AlCalzone — APPROVED** [review]
LGTM unless @blhoward2 has something to add?

**Review by blhoward2 — APPROVED** [review]

**Pull request merged**